docs/src/main/sphinx/release/release-458.md

```diff
@@ -0,0 +1,100 @@
+# Release 458 (13 Sep 2024)
+
+## General
+
+* Improve performance for queries with a redundant `DISTINCT` clause. ({issue}`23087`)
+
+## Druid connector
+
+* Reduce data transfer from remote systems for queries with large `IN` lists. ({issue}`23381`)
+
+## Delta Lake connector
+
+* {{breaking}} Deactivate [legacy file system support](file-system-legacy) for
+  all catalogs. You must activate the desired [file system
+  support](file-system-configuration) with
+  `fs.native-azure.enabled`,`fs.native-gcs.enabled`, `fs.native-s3.enabled`, or
+  `fs.hadoop.enabled` in each catalog. Use the migration guides for [Azure
+  Storage](fs-legacy-azure-migration), [Google Cloud
+  Storage](fs-legacy-gcs-migration), and [S3](fs-legacy-s3-migration) to assist
+  if you have not switched from legacy support. ({issue}`23343`)
+* Add JMX monitoring to the native S3 file system support. ({issue}`23177`)
```

Comment:
"support" at the end of the sentence sounds weird. Rephrase either as "Add support for JMX monitoring to ...", or simply remove the word "support" at the end.

Reply:
Changed to link